index.js

```diff
@@ -89,11 +117,6 @@ function getOptions(uri, o, method) {
         o.timeout = 2 * 60 * 1000; // 2 minutes
     }
 
-    // Default pool options: Don't limit the number of sockets
-    if (!o.pool) {
-        o.pool = {maxSockets: Infinity};
-    }
-
```

Comment:
@arlolra The #21 is doing all the same as this one, but it doesn't do this piece. I'm not quite sure why did you remove this? Could you give some context please?